plugins/block-extension-tooltip/src/index.ts

```diff
@@ -8,7 +8,7 @@
  * @fileoverview Custom tooltip block extension.
  * @author samelh@google.com (Sam El-Husseini)
  */
-import * as Blockly from 'blockly/core';
```

Comment:
Importing the entire `blockly` module would mean including the JS generator and built-in blocks when a user may not need them.
This package doesn't depend on the JS generator bring there, hence why it was only importing `blockly/core`

Reply:
Yeah, it's not ideal. But this is inexplicably causing one of the errors. If I change both the src and test files to use `blockly/core` then I get this error:

```
ERROR in /Users/maribethb/github/maribethb/blockly-samples/plugins/block-extension-tooltip/test/index.ts(79,53):
TS2345: Argument of type '(blocklyDiv: HTMLElement, options: import("/Users/maribethb/github/maribethb/blockly-samples/plugins/block-extension-tooltip/node_modules/blockly/blockly").BlocklyOptions) => import("/Users/maribethb/github/maribethb/blockly-samples/plugins/block-extension-tooltip/node_modules/blockly/blockly").WorkspaceSvg' is not assignable to parameter of type '(blocklyDiv: HTMLElement, options: import("blockly_options").BlocklyOptions) => import("workspace_svg").WorkspaceSvg'.
  Type 'WorkspaceSvg__Class' is missing the following properties from type 'WorkspaceSvg': metricsManager_, componentManager_, blockDragSurface_, workspaceDragSurface_, and 47 more.
```

which is very similar to the error that you get if you leave them how they are (`blockly/core` in src and `blockly` in test) but just on a different line. I'm not sure why this occurs. dev-tools is already using `blockly/core` so it's not just purely that they don't match. Also the .d.ts file in dev-tools says the type is `Workspace` rather than `WorkspaceSvg` so I tried updating that but it doesn't fix this error.

So, yeah... I don't know why this only works if you import from `blockly` instead of `blockly/core` but empirically that's what I'm seeing. It's not ideal that it bloats the module size but it's better than not building at all.

I'm going to work on fixing the plugin so that is is compatible with v7 soon, so this should be temporary (also no idea why this error goes away if the versions of blockly match...). I can add a todo comment with a link back to the issue to remind me to change this back once we don't need it.

Reply:
Oh sorry, I just re-read your comment below and understood what you meant. Unfortunately the "paths" change only fixes the "conflicting files" error and doesn't have any effect on this error. I did just test it to see if that would work but no such luck.